### ISSUE TITLE
Broken GET/POST /courses/:id/students

### ISSUE THREAD

**Issue by TravisWhitehead** [osu-cs493-sp19/final-project-mohamed#6]
@jcorme Any idea what the problem is here? It's a static method so IDK why this wouldn't work calling it on the class.
```
api_1       | TypeError: Cannot read property 'all' of undefined
api_1       |     at Course.enrollments (/usr/src/app/models/course.js:12:32)
api_1       |     at router.get (/usr/src/app/api/courses.js:146:35)
api_1       |     at processTicksAndRejections (internal/process/task_queues.js:86:5)
api_1       | GET /courses/2/students 500 4.497 ms - 38
```
https://github.com/osu-cs493-sp19/final-project-mohamed/blob/0a1920a808b2c86534ca4773360ceffdea9c69d3/models/course.js#L12

**Comment by nhcj** [comment]
It's a circular dependency issue. Not sure why I didn't catch this before. Working on a fix...

**Comment by TravisWhitehead** [comment]
Fixed by #8
